exec/src/klio_exec/commands/run.py

```diff
@@ -552,6 +554,42 @@ def _setup_pipeline(self, pipeline):
                     **output_config.to_io_kwargs()
                 )
 
+    def wait_for_pipeline_running(
+        self,
+        result: PipelineResult,
```

Comment:
I 100% agree with the use of typing. However, Klio does not use typing anywhere, so this would be an anomaly. I'm fine with just `result`, but if you want, maybe renaming this arg to `pipeline_result` would do to help with the lack of typing?

Reply:
👍🏾